lib/tests/fixtures/facts/external/windows/powershell/json.ps1

```diff
@@ -0,0 +1,22 @@
+if (Get-Command ConvertTo-JSON -errorAction SilentlyContinue)
+{
+    Write-Output(@{
```

Comment:
I don't believe calls to `Write-Output` should be necessary here... it should happen automatically.